### PR TITLE
Support viewing container instance logs

### PR DESCRIPTION
**Pull request by lawrencegripper**
#104

### CODE REVIEW

**Comment by stuartleeks** [comment]
Nice!!

I just tried it on an ACI instance with quite large logs and the perf of scrolling the list wasn't great. Looking at the [docs](https://docs.microsoft.com/en-us/rest/api/container-instances/container/listlogs) there is a `tail` parameter - I wonder if we should consider setting a value for that (without it we get the full logs up to 4MB!)

Also, it really makes me want page up/down support in the details view :-)

**Comment by lawrencegripper** [comment]
@stuartleeks hopefully this now behaves a bit better, I've limited the logs to 400 lines and improved the formatting logic.

**Comment by stuartleeks** [comment]
I don't have the previous ACI instance to test against but just created one with a reasonable log size and perf definitely seemed better :-)

**Comment by lawrencegripper** [comment]
Nice, working with this a lot last night I CAN'T ship without pageup/pagedown as it's nighmareish to use. I'm calling time on this flakey test as it's doing my head in. 

Will try and get those fixes in then can look to merge.

**Comment by lawrencegripper** [comment]
Think this is good to go now, anyone able to do a quick review for me?

File: pkg/armclient/types.go [hunk 1]
```diff
@@ -285,3 +285,78 @@ type ActivityLogResource struct {
...
 	} `json:"value"`
 }
+
+// ContainerLogResponse for container logs
+type ContainerLogResponse struct {
+	Content string `json:"content"`
+}
+
+// ContainerGroupResponse is the response to a get request on a container group
+type ContainerGroupResponse struct {
```
Comment (stuartleeks) [inline pkg/armclient/types.go]: Do we want to put these types in here? Could become quite large as we add more data plane functionality.

Comment (lawrencegripper) [reply pkg/armclient/types.go]: Yeah I agree, I'll track an issue for re-orging this stuff in a separate PR.

**Pull request merged**